string guid
string pickled_coord_and_image
---
